CMakeLists.txt

```diff
@@ -57,12 +57,12 @@ else(WIN32)
         -fno-builtin
         -w
         -g3
-        -m32
+        #-m32
     )
 
     # Link everything 32-bit (until we have a 64-bit option)
-    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -m32")
-    set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -m32")
+    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ")#-m32")
+    set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} ") #-m32")
```

Comment:
Could you please add an option to compile 32 bit or 64 bit

Reply:
If it's as easy as making this flag optional, yes!
However, I might also have added compiler flags that 32-bit compilers or let's say older compiler versions, do not handle. Need to look in to it, but will do :+1: